regression/storage.js

```diff
@@ -42,12 +42,12 @@ var files = {
 };
 
 function deleteFiles(bucket, callback) {
-  bucket.getFiles(function(err, files) {
+  bucket.getFiles(function (err, files) {
```

Comment:
Are these single-space changes a result of your editor? They are unnecessary as far as our coding styles are concerned and should be reverted for consistency.